Ballerina.gitignore

```diff
@@ -0,0 +1,20 @@
+# vscode files
+.vscode/
+
+# intellij files
+.idea/
+
+# mac
+.DS_Store
```

Comment:
Would you be open to removing these lines? These are handled by [globals](https://github.com/github/gitignore/blob/main/Global/README.md).

We prefer to keep the templates very tightly focused on the language/tool in question.

Reply:
Absolutely! thanks for pointing that out. I have made the relevant changes in [3af7bb0](https://github.com/github/gitignore/pull/4431/commits/3af7bb0480965c24777d19d8aa6fbfd92080b92a). Pls take a look